src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -382,7 +382,6 @@
         // Clipboard Integration
         { "command": { "action": "copy", "singleLine": false }, "keys": "ctrl+shift+c" },
         { "command": { "action": "copy", "singleLine": false }, "keys": "ctrl+insert" },
-        { "command": { "action": "copy", "singleLine": false }, "keys": "enter" },

```

Comment:
now users cannot unbind it, and it no longer works for mouse selection.........?

Reply:
Correct. I mean, I see the argument for both sides, so I'm not sure what the right answer is.

## Enter as a key binding
Pros...
- Configurable (aka, unbindable)
- works with mouse selection "out-of-the-box"

Other notes...
- we should probably bind <kbd>Shift+Enter</kbd> to copy w/ singleLine = false then? For consistency with mouse-copy?
- we should probably bind <kbd>Ctrl+Enter</kbd> to a new action like "open hyperlink"?

## Enter as a mark mode binding
Pros...
- these features are explicitly a part of "mark mode"

Other notes...
- we're _technically_ taking key chords away from the user, but a selection is present so key chords are designed to clear the selection anyways... so not really?

## Enter as a "that's just how things work" binding
(aka we make it so that "Enter" and "Shift+Enter" copy when a selection is active... always... regardless of how the selection was made)

Pros...
- consistency

Cons...
- no configurability

Thoughts?

Reply:
both-both-is-good.gif